src/page-weekly-snapshot/LocaleSummaryTable.tsx

```diff
@@ -0,0 +1,368 @@
+import { sum } from "d3-array";
+import { get, keys, omit, orderBy, pick, values } from "lodash";
+import React from "react";
+import styled from "styled-components";
+
+import Colors from "../design-system/Colors";
+import { Column, PageContainer } from "../design-system/PageColumn";
+import { useFacilities } from "../facilities-context";
+import {
+  caseBracketKeys,
+  deathBracketKeys,
+  incarceratedPopulationKeys,
+} from "../impact-dashboard/EpidemicModelContext";
+import { formatThousands } from "../impact-dashboard/ImpactProjectionTable";
+import { LocaleData, useLocaleDataState } from "../locale-data-context";
+import { Facility } from "../page-multi-facility/types";
+
+const Table = styled.table`
+  color: ${Colors.black};
+  text-align: left;
+  width: 100%;
+  margin-top: 10px;
+`;
+
+const HorizontalRule = styled.hr<{ width?: string; marginLeft?: string }>`
+  border-color: ${Colors.black};
+  width: ${(props) => props.width || "100%"};
+  margin-left: ${(props) => props.marginLeft || "0"};
+`;
+
+const TableHeadingCell = styled.td`
+  font-family: "Libre Franklin";
+  font-weight: bold;
+  font-size: 11px;
+  line-height: 13px;
+  vertical-align: middle;
+`;
+
+const LeftHeading = styled.div`
+  text-align: left;
+`;
+
+const TextContainerHeading = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 10px;
+`;
+
+const TextContainer = styled.div`
+  width: 100%;
+  margin: 15px 0 15px;
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+  color: ${Colors.black};
+`;
+
+const Right = styled.div`
+  text-align: right;
+`;
+
+const Left = styled.div`
+  text-align: left;
+  font-size: 24px;
+  font-family: "Libre Baskerville";
+`;
+
+const BorderDiv = styled.div`
+  border-top: 2px solid ${Colors.black};
+`;
+
+const TableCell = styled.td<{ label?: boolean }>`
+  font-size: 11px;
+  line-height: 200%;
+  text-align: "left";
+  width: ${(props) => (props.label ? "200px" : "auto")};
+`;
+
+type StateMetrics = {
+  stateName: string;
+  casesPerCapita: number | undefined;
+  deathsPerCapita: number | undefined;
+};
+
+type incarceratedData = {
+  incarceratedData: number;
+  hasData: boolean;
+};
+
+const POPULATION_SIZE = 100000;
+const NUM_STATES = 50;
+
+const VALUE_MAPPING = {
+  deaths: deathBracketKeys,
+  cases: caseBracketKeys,
+  population: incarceratedPopulationKeys,
+};
+
+function formatOrdinal(value: number) {
+  let suffix = ["th", "st", "nd", "rd"],
+    rem = value % 100;
+  return value + (suffix[(rem - 20) % 10] || suffix[rem] || suffix[0]);
+}
+
+function getPerCapita(
+  numerator: number | undefined,
+  denominator: number | undefined,
+) {
+  let result = 0;
+  if (numerator && denominator) {
+    result = Math.round((numerator / denominator) * POPULATION_SIZE);
+  }
+  return result;
+}
```

Comment:
sorry to be pedantic but some of the naming here is a bit confusing to me ... _per capita_ means "per person" but that is not what you are calculating. You could call it `getPer100K` and then `POPULATION_SIZE` is no longer a magic number and doesn't need a constant. Or you could call it `getRate` or something like that and call the constant something a little less vague like `RATE_UNIT` or `RATE_MULTIPLIER`?

(same general pedantry applies to other uses of "per capita" in this PR)